static/app/utils/replays/hooks/useReplayData.tsx

```diff
@@ -22,7 +23,7 @@ type State = {
    */
   fetchingAttachments: boolean;
   fetchingErrors: boolean;
-  fetchingReplay: boolean;
+  // fetchingReplay: boolean;
```

Comment:
Will there be no loading state at all?

Reply:
There's still al loading state!
This value got moved out of the state that's manually tracked, and it's replaced by the return of `useApiQuery`.

`const {... , error: fetchReplayError} = useApiQuery(...)` is the new name for the same value, down a bit lower.

Reply:
Let's remove the commented code

Reply:
d'oh